cf-deployment.yml

```diff
@@ -168,24 +168,24 @@ instance_groups:
           name: etcd
         peer_require_ssl: true
         require_ssl: true
-        ca_cert: "((etcd_ca_cert))"
-        client_cert: "((etcd_client_cert))"
-        client_key: "((etcd_client_key))"
-        server_cert: "((etcd_server_cert))"
-        server_key: "((etcd_server_key))"
-        peer_ca_cert: "((etcd_peer_ca_cert))"
-        peer_cert: "((etcd_peer_cert))"
-        peer_key: "((etcd_peer_key))"
+        ca_cert: "((etcd_client.ca))"
```

Comment:
Shouldn't this just be `((etcd.ca))`? It's used to sign both the client and the server certs, isn't it?